cyclops/tasks/utils.py

```diff
@@ -108,9 +108,7 @@ def prepare_models(
         models_dict = {model_name: models}
     # models contains one model name
     elif isinstance(models, str):
-        assert (
-            models in list_models()
-        ), f"Model name is not registered! \
+        assert models in list_models(), f"Model name is not registered! \
```

Comment:
Not related to this PR, but it's usually a good idea to raise an exception instead of asserting; asserts are optimized away if the interpreter is running with optimizations enabled.